engine/resources/mount.go

```diff
@@ -243,36 +323,36 @@ func (obj *MountRes) Watch(ctx context.Context) error {
 	conn.Signal(ch)
 	defer conn.RemoveSignal(ch)
 
-	// watch the fstab file
-	recWatcher, err := recwatch.NewRecWatcher(fstabPath, false)
-	if err != nil {
-		return err
-	}
-	// close the recwatcher when we're done
-	defer recWatcher.Close()
-
 	obj.init.Running() // when started, notify engine that we're running
 
+	// // watch the unit file
```

Comment:
This should probably be uncommented, but I want better end-to-end testing facilities first. These wrapped resources make life a little bit complicated.